quality/extract_cci_approved_executed.py

```diff
@@ -72,6 +72,14 @@ def get_cci_metadata(prune=False):
 
 metadata_df = get_cci_metadata()
 merged_metadata_df = pd.merge(metadata_df, existing_metadata_df, on=['country_code'], how='left', suffixes=('', '_old'))
+
+# handling for the new countries added
+existing_countries = existing_metadata_df.country.unique()
+current_countries = metadata_df.country.unique()
+new_countries = [country for country in current_countries if country not in existing_countries]
+for country in new_countries:
+    merged_metadata_df.loc[merged_metadata_df.country == country, 'updated_at_old'] =  existing_metadata_df.loc[merged_metadata_df.country == country,'updated_at'] - 1e+09
+
```

Comment:
@yukinko-iwasaki You are right about the new country case not being covered by the previous code. The added code does the job but I think it can be simplified by using `fillna`, because only new countries will have na for the `updated_at_old` field. Something like:
```
older_updated_at = merged_metadata_df['updated_at'] - 1e+09
merged_metadata_df['updated_at_old'].fillna(older_updated_at, inplace=True)
```